### PR TITLE
Fixing orientation, enlarging color segmented control, scroll view work.

### PR DESCRIPTION
**Pull request by piperchester**
Still need to re-implement the scroll view system. Have made significant progress though.

### CODE REVIEW

File: AccessLecture/LectureViewController.h [hunk 1]
```diff
@@ -20,13 +21,15 @@
     
     // Image grabbing
     NSMutableData* receivedData;
-    NSTimer* t;
     UIImage* img;
     
     // Zooming
-    CGPoint scrSize;
+    CGPoint screenSize;
```
Comment (7imbrook) [inline AccessLecture/LectureViewController.h:27]: When using a size, should be using a CGSize (has height and width) it's more descriptive of what you're trying to do. Good practice

Comment (piperchester) [inline AccessLecture/LectureViewController.h:27]: Good point (I'm so punny).

File: AccessLecture/LectureViewController.m [hunk 2]
```diff
@@ -341,18 +245,28 @@ - (void)scrollViewDidEndZooming:(UIScrollView *)sv withView:(UIView *)view atSca
 
 #pragma mark - Zooming 
 
+- (void)zoomTap:(UIGestureRecognizer *)gestureRecognizer {
+    NSLog(@"Zoom Tap!");
```
Comment (7imbrook) [inline AccessLecture/LectureViewController.m]: No logging on master -__- (We should make a logging method that can be defined to be on/off at compile time (Using debug mode (and nested parenthetic statements))).

Comment (piperchester) [inline AccessLecture/LectureViewController.m]: Ooh good find. Completely forgot about that log! Totally agree.

**Pull request merged**